crates/cli/src/benchmark.rs

```diff
@@ -244,8 +244,14 @@ impl BenchmarkCommand {
             .with_context(|| "expected the benchmark file to have an extension")?
             .to_str()
             .with_context(|| "expected the benchmark file to have a printable name")?;
-        let stdout_expected = wasm_file_dir.join(format!("{}.stdout.expected", benchmark_name));
-        let stderr_expected = wasm_file_dir.join(format!("{}.stderr.expected", benchmark_name));
+        let mut stdout_expected = wasm_file_dir.join(format!("{}.stdout.expected", benchmark_name));
+        if !stdout_expected.exists() {
+            stdout_expected = wasm_file_dir.join(format!("{}.stdout.expected", "default"));
+        }
+        let mut stderr_expected = wasm_file_dir.join(format!("{}.stderr.expected", benchmark_name));
+        if !stderr_expected.exists() {
+            stderr_expected = wasm_file_dir.join(format!("{}.stderr.expected", "default"));
```

Comment:
```suggestion
            stderr_expected = wasm_file_dir.join("default.stderr.expected");
```

Reply:
@abrown .. the code here is fine but I was under the impression that all the libsodium benchmarks returned 0 but some actually don't write anything to the log. Below is a cat of some of the logs when running all benchmarks in the suite:

```
stdout-d3b16d56d6c3a977-691414-2.log
::::::::::::::
0
::::::::::::::
stdout-d665976133d9c65-691414-0.log
::::::::::::::
0
::::::::::::::
stdout-d665976133d9c65-691414-1.log
::::::::::::::
0
::::::::::::::
stdout-d665976133d9c65-691414-2.log
::::::::::::::
0
::::::::::::::
stdout-d6c7b658a5a51c0a-691414-0.log
::::::::::::::
::::::::::::::
stdout-d6c7b658a5a51c0a-691414-1.log
```

To avoid modifications, I can just add empty expected output files for the ones that return nothing. But before I do that I'll look to see if it makes sense to return 0 so that all files share the same default.stdout.expected and default.stderr.expected files.

Reply:
@abrown Resolved with the latest push to include a couple of non-default expected files.

Reply:
It's good that you resolved the CI issues but that is not what this comment was originally about: there's no reason to generate a string with `format!` when we already know what the string value should be.